packages/twenty-server/src/modules/activity/standard-objects/activity.workspace-entity.ts

```diff
@@ -40,20 +51,25 @@ export class ActivityWorkspaceEntity extends BaseWorkspaceEntity {
 
   @WorkspaceField({
     standardId: ACTIVITY_STANDARD_FIELD_IDS.body,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.RICH_TEXT,
```

Comment:
If this does not impact the data this is fine BUT we need a command to update all existing fieldMetadata

Reply:
Isn't that what sync-metadata should do?